### PR TITLE
chore(mixin): Update dashboards for bloom components

### PR DESCRIPTION
**Pull request by chaudum**
**What this PR does / why we need it**:

* Update bloom gateway dashboard 
* Replace bloom compactor with bloom build dashboard

**Special notes for your reviewer**:

Followed the instructions

```
// !--- HOW TO UPDATE THIS DASHBOARD ---!
// 1. Export the dashboard from Grafana as JSON
//    !NOTE: Make sure you collapse all rows but the (first) Overview row.
// 2. Copy the JSON into `dashboard-bloom-xxx.json`
// 3. Delete the `id` and `templating` fields from the JSON
```

**Checklist**
- [x] Reviewed the [`CONTRIBUTING.md`](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) guide (**required**)
- [ ] Documentation added
- [ ] Tests updated
- [x] Title matches the required conventional commits format, see [here](https://www.conventionalcommits.org/en/v1.0.0/)
  - **Note** that Promtail is considered to be feature complete, and future development for logs collection will be in [Grafana Alloy](https://github.com/grafana/alloy). As such, `feat` PRs are unlikely to be accepted unless a case can be made for the feature actually being a bug fix to existing behavior.
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/setup/upgrade/_index.md`
- [ ] If the change is deprecating or removing a configuration option, update the `deprecated-config.yaml` and `deleted-config.yaml` files respectively in the `tools/deprecated-config-checker` directory. [Example PR](https://github.com/grafana/loki/pull/10840/commits/0d4416a4b03739583349934b96f272fb4f685d15)

### CODE REVIEW

File: production/loki-mixin-compiled-ssd/dashboards/loki-bloom-gateway.json [hunk 1]
```diff
@@ -737,44 +714,941 @@
...
+                        "editorMode": "code",
+                        "expr": "sum(rate(loki_bloom_recorder_series_total{cluster=\"$cluster\", job=\"$namespace/bloom-gateway\", type=\"filtered\"}[$__rate_interval])) by (type)\n/ on () group_left\nsum(rate(loki_bloom_recorder_series_total{cluster=\"$cluster\", job=\"$namespace/bloom-gateway\", type=\"found\"}[$__rate_interval]))",
+                        "hide": false,
+                        "instant": false,
+                        "legendFormat": "__auto",
+                        "range": true,
+                        "refId": "B"
+                     }
+                  ],
+                  "title": "Filtered series",
```
Comment (rfratto) [inline production/loki-mixin-compiled-ssd/dashboards/loki-bloom-gateway.json:1407]: This panel was blank for me, while "Found/Skipped/Missing series" wasn't. Bug?

File: production/loki-mixin-compiled-ssd/dashboards/loki-bloom-gateway.json [hunk 2]
```diff
@@ -1662,22 +2655,37 @@
...
+                        "expr": "sum(sum by(instance) (rate(node_disk_written_bytes_total[$__rate_interval]))\n+ on(instance) group_right() \n(count by(instance) (container_fs_writes_bytes_total{cluster=~\"$cluster\", namespace=~\"$namespace\", container=\"bloom-gateway\", device!~\".*sda.*\"}) * 0)) * -1",
+                        "hide": false,
+                        "instant": false,
+                        "legendFormat": "Writes",
+                        "range": true,
+                        "refId": "A"
                      }
                   ],
-                  "title": "Disk writes",
+                  "title": "Disk reads/writes",
```
Comment (rfratto) [inline production/loki-mixin-compiled-ssd/dashboards/loki-bloom-gateway.json:2688]: With a 12h time range, this fails with 

```
execution: query processing would load too many samples into memory in query execution
``` 

It works with a 1h time range though. Should we change the query to load less samples?

File: production/loki-mixin-compiled-ssd/dashboards/loki-bloom-gateway.json [hunk 3]
```diff
@@ -4799,180 +5784,153 @@
...
-                        "legendFormat": "Size",
+                        "instant": false,
+                        "legendFormat": "__auto",
                         "range": true,
                         "refId": "B"
                      }
                   ],
-                  "title": "Block download queue",
-                  "type": "timeseries"
+                  "title": "Dequeue count",
```
Comment (rfratto) [inline production/loki-mixin-compiled-ssd/dashboards/loki-bloom-gateway.json:5924]: This panel is blank for me even though "Dequeue duration" isn't blank. Bug?

**Review by rfratto — APPROVED** [review]
I found a few minor issues in the dashboards but otherwise LGTM. I only left my comments on the compiled-ssd mixin, but I think what I found probably applies to all copies of them.

File: production/loki-mixin-compiled-ssd/dashboards/loki-bloom-gateway.json (hunk 3 above)
Comment (chaudum) [reply production/loki-mixin-compiled-ssd/dashboards/loki-bloom-gateway.json:5924]: Interesting, the metric definitely exists https://github.com/grafana/loki/pull/14413

File: production/loki-mixin-compiled-ssd/dashboards/loki-bloom-gateway.json (hunk 1 above)
Comment (chaudum) [reply production/loki-mixin-compiled-ssd/dashboards/loki-bloom-gateway.json:1407]: This seems to be a bug in Loki where we don't record series any more:

```
sum(rate(loki_bloom_recorder_series_total{cluster="$cluster", job="$namespace/bloom-gateway", type="filtered"}[$__rate_interval]))
```
Does not yield anything, where else
```
sum(rate(loki_bloom_recorder_chunks_total{cluster="$cluster", job="$namespace/bloom-gateway", type="filtered"}[$__rate_interval]))
```
does

**Pull request merged**